src/frontend/js/components/ReferralDetail/ChangeUrgencyLevelModal.tsx

```diff
@@ -120,6 +120,9 @@ export const ChangeUrgencyLevelModal: React.FC<ChangeUrgencyLevelModalProps> = (
       ariaHideApp={!isTestEnv}
       isOpen={isChangeUrgencyLevelModalOpen}
       onRequestClose={() => setIsChangeUrgencyLevelModalOpen(false)}
+      onAfterOpen={() => {
+        setNewUrgencyLevelId(data!.results[0]['id']);
```

Comment:
I think this is not the right way to do this. I have two proposals:

1) it's correct to default to the current value for the referral urgency, but it's the `<select>` that needs to reflect this. Also, then, it should not be possible to validate the form when the newUrgencyLevel is the same as what's already on the referral.

2) we could simply not display the current urgency in the select, and then show a message above the select with eg. "Current urgency: 3 weeks" so the user understands it's not just by mistake.

What do you think?